lib/puppet/type/cisco_bgp_af.rb

```diff
@@ -196,6 +198,34 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end # property :default_information_originate
 
+  newproperty(:maximum_paths) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
+          Valid values are integers between 1 and 64, default value is 1."
+    munge do |value|
+      value = :default if value == 'default'
+      unless value == :default
+        value = value.to_i
+        fail 'maximum_paths value should be between 1 and 64' unless
+          value.between?(1, 64)
+      end
+      value
+    end
+  end # property :maximum_paths
+
+  newproperty(:maximum_paths_ibgp) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
+          Valid values are integers between 1 and 64, default value is 1."
```

Comment:
/Valid values are integers between 1 and 64/Valid value is an integer in the range 1 - 64/